lib/contract/test/api.go

```diff
@@ -0,0 +1,101 @@
+package test
+
+import (
+	"fmt"
+
+	"boscoin.io/sebak/lib/contract/context"
+	"boscoin.io/sebak/lib/contract/payload"
+	"boscoin.io/sebak/lib/contract/storage"
+	"boscoin.io/sebak/lib/contract/value"
+)
+
+type (
+	HelloworldFunc     func(context.Context, string) (string, error)
+	GetStorageItemFunc func(context.Context, string) (*storage.StorageItem, error)
+	PutStorageItemFunc func(context.Context, string, *storage.StorageItem) error
+	CallContractFunc   func(context.Context, *payload.ExecCode) (*value.Value, error)
+)
```

Comment:
It seems looks ugly. but the pattern that the MockAPI use is simple. 
It looks like ugly DI.